common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -100,8 +101,13 @@ function(HTML5Video, Resizer) {
     //     initial configuration. Also make the created DOM elements available
     //     via the 'state' object. Much easier to work this way - you don't
     //     have to do repeated jQuery element selects.
+    // eslint-disable-next-line no-underscore-dangle
     function _initialize(state) {
-        var youTubeId, player, userAgent;
+        var youTubeId,
+            player,
+            userAgent,
+            commonPlayerConfig,
+            eventToBeTriggered = 'loadedmetadata';
```

Comment:
`loadedmetadata` is the name of actual event we want to trigger. We can't change this.